test/contlinear.jl

```diff
@@ -1063,6 +1063,45 @@ function linear11test(solver::MOI.AbstractSolver; atol=Base.rtoldefault(Float64)
     end
 end
 
+function linear12test(solver::MOI.AbstractSolver; atol=Base.rtoldefault(Float64), rtol=Base.rtoldefault(Float64))
+    @testset "test infeasible problem with 2 linear constraints" begin
+        # min x
+        # s.t. 2x-3y <= -7
+        #      y <= 2
+        # x,y >= 0
+        @test MOI.supportsproblem(solver, MOI.ScalarAffineFunction{Float64}, [(MOI.ScalarAffineFunction{Float64},MOI.GreaterThan{Float64}),(MOI.SingleVariable,MOI.GreaterThan{Float64})])
+
+        m = MOI.SolverInstance(solver)
+        x = MOI.addvariable!(m)
+        y = MOI.addvariable!(m)
+        c1 = MOI.addconstraint!(m, MOI.ScalarAffineFunction([x,y], [2.0,-3.0], 0.0), MOI.LessThan(-7.0))
+        c2 = MOI.addconstraint!(m, MOI.ScalarAffineFunction([y], [1.0], 0.0), MOI.LessThan(2.0))
+        bndx = MOI.addconstraint!(m, MOI.SingleVariable(x), MOI.GreaterThan(0.0))
+        bndy = MOI.addconstraint!(m, MOI.SingleVariable(y), MOI.GreaterThan(0.0))
+        MOI.set!(m, MOI.ObjectiveFunction(), MOI.ScalarAffineFunction([x], [1.0], 0.0))
+        MOI.set!(m, MOI.ObjectiveSense(), MOI.MinSense)
+        MOI.optimize!(m)
+
+        @test MOI.canget(m, MOI.ResultCount())
+        if MOI.get(m, MOI.ResultCount()) >= 1
+            # solver returned an infeasibility ray
+            @test MOI.get(m, MOI.TerminationStatus()) == MOI.Success
+            @test MOI.get(m, MOI.DualStatus()) == MOI.InfeasibilityCertificate
+            @test MOI.canget(m, MOI.ConstraintDual(), c1)
+            cd1 = MOI.get(m, MOI.ConstraintDual(), c1)
+            cd2 = MOI.get(m, MOI.ConstraintDual(), c2)
+            bndyd = MOI.get(m, MOI.ConstraintDual(), bndy)
+            @test (cd2-bndyd)/cd1 ≈ 3 atol=atol rtol=rtol
```

Comment:
Could you add tests for the signs of the dual values ?
I would also rather write `- 3 cd1 + cd2 ≈ bndyd` and `2 cd1 ≈ bndxd` to make it clearer why the test should be true by being closer to the dual constraint.

Reply:
We could also add a test for the objective `-7a + 2b > 0`